test/functional/services/lib/leadfoot_element_wrapper.js

```diff
@@ -86,6 +86,17 @@ export class LeadfootElementWrapper {
     await this._leadfootElement.type(value);
   }
 
+  /**
+   * Sends keyboard event into the element.
+   * https://theintern.io/leadfoot/module-leadfoot_Session.html#pressKeys
+   *
+   * @param  {string|string[]} keys
+   * @return {Promise<void>}
+   */
+  async pressKeys(...args) {
+    await this._leadfoot.pressKeys(...args);
```

Comment:
Hmm, this isn't an element specific method, why put it on the element wrapper? Feels like it should be in the browser service unless it's going to become element specific when we move to webdriver. In that case though we should remove the `browser.pressKeys()` function.

Reply:
Per webdriver spec it is an[ element method](https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElement.html#sendKeys): with webdriver keyboard events can be sent only on element object, tests are failing. This is the reason why we need it - compatibility for webdriver